tests/foreman/__init__.py

```diff
@@ -0,0 +1,3 @@
+from robottelo.config import settings
+
+settings.configure()
```

Comment:
@Ichimonji10 we can move this call to every test case class, but doing this here will avoid the need of calling this for every single test case. But, the idea behind the design is to call configure everytime that you need robottelo stuff, if already configured.

To make easier, as most of the test cases inherit from robottelo.test.TestCase (API, CLI and UI subclasses do that) we can move this call to there instead at the setUp[Class] method.